docOps/docs/libraries/python/examples.md

```diff
@@ -0,0 +1,756 @@
+
+The storage is encrypted at rest, so you need a strong password and location where to place your storage. 
+
+:::warning
+We highly recommended that you to store your `Stronghold` password encrypted on rest and separated from `Stronghold` snapshots. 
+
+Deal with the password with utmost care.
+:::
+
+ The storage comprises two things:
+* A single file called _wallet.stronghold_ , which contains _seed_ and is secured by `Stronghold` and encrypted at rest. The generated seed (mnemonic) serves as a cryptographic key from which all accounts and related addresses are generated.
+* Other data used by library that is stored under the _db_ sub-directory.  The includes account information, generated addresses, fetched messages, etc.  This data is used to speed up some operations, such as account creation, address generation, etc.
+
+One of the key principles behind `Stronghold` based storage is that no one can extract a seed from the storage. You deal with all accounts purely via an _AccountManager_ instance.  All complexities are hidden under the hood and dealt with securely.
+
+If you also want to store a seed somewhere else, you can use the _AccountManager.generateMnemonic()_ method. This method will generate a random seed, and it can be used before the actual account initialization.
+
+You can find detailed information about seed generation at [Developer Guide to Chrysalis](https://chrysalis.docs.iota.org/guides/dev_guide.html#seed).
+
+### Accounts
+The `Wallet.rs` library uses a model of individual accounts to separate individual users/clients from each other. It is possible to generate multiple addresses for each account deterministically. 
+
+Each account is related to a specific IOTA network (mainnet / testnet), which is referenced by node properties such as node url.  In this example, the `Chrysalis` testnet balancer.
+
+For more information about _client_options_ , please refer to [Wallet Python API Reference](api_reference.md#clientoptions).
+
+
+```python
+# ... continue from prev example 1a
+
+# general Tangle specific options
+client_options = {
+    "nodes": [
+        {
+            "url": "https://api.hornet-0.testnet.chrysalis2.com",
+            "auth": None,
+            "disabled": False
+        }
+    ],
+    "local_pow": True
+}
+
+# an account is generated with the given alias via `account_initialiser`
+account_initialiser = account_manager.create_account(client_options)
+account_initialiser.alias('Alice')
+
+# initialise account based via `account_initialiser`
+# store it to db and sync with Tangle
+account = account_initialiser.initialise()
+print(f'Account created: {account.alias()}')
+```
+`Alias` should be unique, and it can be any string that you see fit. The _alias_ is usually used to identify the account later on. Each account is also represented by an _index_ which is incremented by 1 every time new account is created. 
```

Comment:
```suggestion
_Alias_ should be unique, and it can be any string that you see fit. The _alias_ is usually used to identify the account later on. Each account is also represented by an _index_ which is incremented by 1 every time new account is created. 
```